ui/admin/app/styles/app.scss

```diff
@@ -1025,13 +1025,29 @@
   text-overflow: ellipsis;
 }
 
-.confirmation-modal {
-  .hds-button-set {
-    margin-bottom: 0;
+.worker-tags {
+  .confirmation-modal {
+    .hds-button-set {
+      margin-bottom: 0;
+    }
+
+    .hds-form-field--layout-vertical {
+      margin-top: 1.5rem;
+      margin-bottom: 0;
+    }
   }
 
-  .hds-form-field--layout-vertical {
+  .edit-modal {
+    .hds-button-set {
+      margin-bottom: 0;
+    }
+
+    // div:nth-child(2) span {
+    //   margin-bottom: 1.5rem;
+    // }
+  }
+
+  .hds-application-state {
     margin-top: 1.5rem;
-    margin-bottom: 0;
```

Comment:
does this change affect anywhere else we use app state component?

Reply:
no this is limited to `.hds-application-state` within the worker/tags route.